### DISCUSSION TITLE
Thank You!

### DISCUSSION THREAD

**Discussion by rjm831** [j-fbriere/squawker#49] [General]
Just a simple thank you to everyone involved in keeping this project alive.  It has got to be massively frustrating dealing with the constant harassment by Twitter.  I truly appreciate the hours you folks spend in maintaining this app.  Bless you all.

**Comment by kingleo1** [comment]
I would love to also thank the Devs of this project!
You are absolute real world heroes. Legends. Twitter is simply unusable without your time and effort to play the cat and mouse game with that billionaire idiot.
You deserve all the best!
Thank you so much!

**Comment by j-fbriere** [comment]
@rjm831 @kingleo1 Thank you.
You kind words are very much appreciated.
